protocol/communities/manager.go

```diff
@@ -67,8 +89,13 @@ func NewManager(identity *ecdsa.PublicKey, db *sql.DB, logger *zap.Logger, verif
 }
 
 type Subscription struct {
-	Community   *Community
-	Invitations []*protobuf.CommunityInvitation
+	Community                      *Community
+	Invitations                    []*protobuf.CommunityInvitation
+	CreatingHistoryArchivesSignal  *signal.CreatingHistoryArchivesSignal
+	HistoryArchivesCreatedSignal   *signal.HistoryArchivesCreatedSignal
+	NoHistoryArchivesCreatedSignal *signal.NoHistoryArchivesCreatedSignal
+	HistoryArchivesSeedingSignal   *signal.HistoryArchivesSeedingSignal
+	HistoryArchivesUnseededSignal  *signal.HistoryArchivesUnseededSignal
```

Comment:
I could build a separate `Subscription` type for this, but I thought there's no reason not to put those events in here.